io-backend/src/main/java/agh/io/iobackend/DataGenerator.java

```diff
@@ -0,0 +1,118 @@
+package agh.io.iobackend;
+
+import agh.io.iobackend.controller.AuthController;
+import agh.io.iobackend.controller.MapController;
+import agh.io.iobackend.controller.payload.SignupRequest;
+import agh.io.iobackend.model.Vector;
+import agh.io.iobackend.model.map.GameMap;
+import agh.io.iobackend.model.map.MapStructure;
+import agh.io.iobackend.service.StatisticsService;
+import agh.io.iobackend.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.ArrayList;
+
+/*
+This class is responsible for generating example data.
+ */
+//@Component // <- comment out if you want to generate some data during application startup
+public class DataGenerator {
+
+    @Autowired
+    private MapController mapController;
+
+    @Autowired
+    private AuthController authController;
+
+    @Autowired
+    private UserService userService;
+
+    @Autowired
+    private StatisticsService statisticsService;
+
+    private final String username1 = "great-user1";
+    private final String email1 = "greatemail1@gmail.com";
+    private final String username2 = "great-user2";
+    private final String email2 = "greatemail2@gmail.com";
+    private final String password = "pass";
+    private Long user1Id;
+    private Long user2Id;
+    private Long map1Id;
+    private Long map2Id;
+
+    @PostConstruct
+    private void generateInitialData() {
+        addUsers();
+        addMaps(user1Id);
+        addGameMapHistory();
+    }
+
+    private void addUsers() {
```

Comment:
Pierdoła, ale można by tą metode zmienić tak, aby dodawała wyłącznie jednego usera, i po prostu byłby odpalalana dla każdego jednego.  Zależy po prostu czy będziemy sklaować to, czy nie, narazie jest git.